examples/third_party/cares/BUILD.bazel

```diff
@@ -1,8 +1,24 @@
-load("@rules_cc//cc:defs.bzl", "cc_test")
+load("@build_bazel_rules_apple//apple:ios.bzl", "ios_build_test")
+load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
 
 cc_test(
     name = "test_c_ares",
     srcs = ["c-ares-test.cpp"],
     visibility = ["//:__pkg__"],
     deps = ["@cares"],
 )
+
+cc_library(
+    name = "ios_lib",
+    srcs = ["c-ares-test.cpp"],
+    tags = ["manual"],
+    deps = ["@cares"],
```

Comment:
Wow - this is simpler than I expected - I would have expected us to need to set `CMAKE_SYSTEM_NAME` in the CMake target but clearly CMake has enough logic to autodetect the correct config from the environment?

Reply:
Yea the key is that CMake respects `SDKROOT` to determine this I think. It works great even in the complex cases with envoy that has some significant rules using this as well